### PR TITLE
Add package reprimand

### PR DESCRIPTION
**Pull request by wokast**
This intends to add a package for the scientific library "reprimand". Reprimand consists of a C++ library and a Python interface. This pull request intends to only create a package for the C++ library. The Python API will be a separate package.

### CODE REVIEW

**Comment by wokast** [comment]
@conda-forge/staged-recipes Ready for review

**Comment by wokast** [comment]
@conda-forge/staged-recipes Since I did not receive any feedback on my review request, I'd like to inquire if there is anything I can do to help?

**Comment by wokast** [comment]
@conda-forge-admin, please ping team

**Comment by ocefpaf** [comment]
Closing and re-opening to toggle the CIs here.

File: recipes/reprimand/build.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+set -e
+
+export BOOST_ROOT=$PREFIX
+export MBUILD_DIR=build
+
+mkdir -p $MBUILD_DIR
+meson setup --prefix=$PREFIX --libdir=lib $MBUILD_DIR
+cd $MBUILD_DIR
+ninja 
+ninja install
```
Comment (ocefpaf) [inline recipes/reprimand/build.sh:12]: Is there a `make check` or test we can run there?

File: recipes/reprimand/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    - test -f $PREFIX/include/reprimand/global_registry.h        
+    - test -f $PREFIX/include/reprimand/star_seq_file.h
+    - test -f $PREFIX/include/reprimand/eos_barotropic_internals.h  
+    - test -f $PREFIX/include/reprimand/hdf5store.h              
+    - test -f $PREFIX/include/reprimand/star_sequence.h
+    - test -f $PREFIX/include/reprimand/eos_barotr_poly.h           
+    - test -f $PREFIX/include/reprimand/hydro_atmo.h             
+    - test -f $PREFIX/include/reprimand/unitconv.h
+    - test -f $PREFIX/include/reprimand/eos_barotr_pwpoly.h         
+    - test -f $PREFIX/include/reprimand/hydro_cons.h
```
Comment (ocefpaf) [inline recipes/reprimand/meta.yaml:77]: This is probably an overkill. I'd keep just the first check.

File: recipes/reprimand/build.sh (hunk 1 above)
Comment (wokast) [reply recipes/reprimand/build.sh:12]: There are unit tests, but they are run via the meson build system, not make. I was unable to figure out if meson is available in the testing CI environment. If using meson for tests is fine, I would also have to activate building the tests via another option to the meson setup line above.

File: recipes/reprimand/meta.yaml (hunk 2 above)
Comment (wokast) [reply recipes/reprimand/meta.yaml:77]: I agree. The install process is straightforward copying and the most likely problem would be issues for the whole install folder.

File: recipes/reprimand/build.sh (hunk 1 above)
Comment (ocefpaf) [reply recipes/reprimand/build.sh:12]: I used "make check" in a generic way. Yes please, try to run the meson tests b/c they are more robust than the "if exist" test below.

Comment (wokast) [reply recipes/reprimand/build.sh:12]: Ok. Followup question: is it fine to run the test in the build script or do I need to run them in the test section of meta.yaml? I know how to do the former, not sure about the latter.

Comment (ocefpaf) [reply recipes/reprimand/build.sh:12]: For this kind of this the build script is the right place. Any "post install" tests should be in the meta.yaml or a `run_test.<ext>`.

**Pull request merged**